C++ package install fails on Windows

Installing a build on Windows produces an odd error when trying to install the .dll:

```
cmake --install .
-- Install configuration: "Release"
-- Up-to-date: C:/dev/cppinstall/lib/cmake/Slint/SlintTargets.cmake
-- Up-to-date: C:/dev/cppinstall/include/slint/slint.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_brush.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_callbacks.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_color.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_config.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_image.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_interpreter.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_pathdata.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_platform.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_point.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_properties.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_sharedvector.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_size.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_string.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_testing.h
-- Up-to-date: C:/dev/cppinstall/include/slint/vtable.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_enums_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_string_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_brush_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_sharedvector_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_properties_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_image_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_color_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_pathdata_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_qt_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_backend_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_generated_public.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_interpreter_internal.h
-- Up-to-date: C:/dev/cppinstall/include/slint/slint_interpreter_generated_public.h
CMake Error at api/cpp/cmake_install.cmake:95 (file):
  file INSTALL cannot find "C:/dev/cppbuild/bin/release/slint_cpp.dll": No
  error.
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)

```

Source dir is `c:\dev\slint`, build dir is `c:\dev\cppbuild`\ and `CMAKE_INSTALL_PREFIX` is `c:\dev\cppinstall`.

I can reproduce this with and without Ninja, but oddly it's not reproducible in the CI's cpack step.

CMake: version 3.24.2

--- a/api/cpp/CMakeLists.txt
+++ b/api/cpp/CMakeLists.txt
@@ -11,7 +11,7 @@ include(FetchContent)
 FetchContent_Declare(
     Corrosion
     GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
-    GIT_TAG v0.3.2
+    GIT_TAG v0.3.3
 )
 FetchContent_MakeAvailable(Corrosion)
 

